features/step_definitions.rb

```diff
@@ -362,8 +362,22 @@
 
 #
 
-Then(%r!^I should see today's time in "(.*)"$!) do |file|
-  step %(I should see "#{seconds_agnostic_time(Time.now)}" in "#{file}")
+Then(%r!^I should see today's time in "(.*)" with (\d+) seconds? tolerance$!) do |file, seconds|
+  seconds = seconds.to_i
```

Comment:
I think five seconds is a perfectly valid amount of tolerance that doesn't need configuration and we can simply hard code that here. Thoughts? We could even lower the allowed time to something like 2 or 3 seconds.

```suggestion
  seconds = 5
```